compiler/accelerators/snax.py

```diff
@@ -144,96 +152,38 @@ def _generate_streamer_setup_vals(
                 cst = arith.Constant.from_int_and_width(stride.data, i32)
                 result.append(([cst], cst.result))
 
-        # spatial strides
-        for operand, streamer in enumerate(self.streamer_config.data.streamers):
-            strides = iter(op.stride_patterns.data[operand].spatial_strides.data)
-            stride: IntAttr | None = next(strides, None)
-            for dim, flag in enumerate(streamer.spatial_dims):
-                if flag == StreamerFlag.Irrelevant:
-                    # Irrelevant spatial strides are not programmed
-                    if stride and stride.data == 0:
-                        stride = next(strides, None)
-                    continue
-                assert stride is not None  # stride should be intattr here
-                cst = arith.Constant.from_int_and_width(stride, i32)
-                result.append(([cst], cst.result))
-                stride = next(strides, None)
-            assert stride is None  # all strides must be processed by this point
-
-        # input & output base pointers
-        result.extend(([], x) for x in op.inputs)
-        result.extend(([], x) for x in op.outputs)
-
         # transpose specifications
         for operand, streamer in enumerate(self.streamer_config.data.streamers):
             if streamer.type is StreamerType.ReaderTranspose:
-                # always program to 0 for now
-                c0 = arith.Constant.from_int_and_width(0, i32)
-                result.append(([c0], c0.result))
-                # in the current version of the streamer,
-                # transpose fields are bit packed together,
-                # so only one field for all transpose ops
-                # this should be fixed in the new streamer
-                break
+                # if we want to disable transpose, we obviously need to program
+                # a 1 to the transpose field (much logical)
+                # threrefore always program to 1
```

Comment:
Although I share the sentiment and appreciate the joke, I'm not a big fan of sarcastic comments in code. Let's keep it very clear for other people as well :)